packages/ogmios/package.json

```
@@ -40,7 +40,7 @@
     "ws": "^8.5.0"
   },
   "dependencies": {
-    "@cardano-ogmios/client": "5.4.0",
+    "@cardano-ogmios/client": "5.5.1",
```

```suggestion
    "@cardano-ogmios/client": "5.5.2",
```

https://github.com/CardanoSolutions/ogmios/releases/tag/v5.5.2